dapla-manual/statistikkere/navnestandard.qmd

```diff
@@ -192,22 +205,83 @@ ssb-dapla-example-data-produkt-prod/
     │   └── ameldingen_p2024-12_v1.parquet
 ```
 
-## Partisjonerte data
+## Versjonering av datasett
 
-Team som partisjonerer sine filer ved lagring skal fortsatt følge navnestandarden. Det som endrer seg er at **filtype** ikke blir en del av filnavnet, men heller kommer inn under partisjoneringen. Anta at team **dapla-example** partisjonerer et datasett i datatilstand inndata som heter **skjema_p2018_p2020_v1**. Anta også at de partisjonerer dataene med hensyn på kolonnen **aar**. Da vil de i henhold til navnestandarden opprette denne strukturen:
+Versjonering er obligatorisk når man jobber med data på dapla. Hovedgrunnen til at vi versjonerer er for å dekke kravet om uforanderlighet og etterprøvbarehet: at data-konsumenter (menneske eller maskin) skal ha kontroll på endringer. Med andre ord versjonerer vi for å ikke slette data. Les mer om prinsippet om uforanderlighet av data på [confluence-siden til IT-Arkitektur](https://statistics-norway.atlassian.net/wiki/spaces/Arkitektur/pages/2839707937/Prinsipp+1+Uforanderlighet+av+data).
```

Comment:
Kan vi endre fra 'Med andre ord versjonerer vi for å ikke slette data.' til 'Derfor skal et datasett som er brukt, direkte eller indirekte, i statistikkproduksjon aldri slettes, det skal opprettes en ny versjon av datasettet.'